torchgeo/datamodules/naip.py

```diff
@@ -41,22 +39,25 @@ def __init__(
         """Initialize a LightningDataModule for NAIP and Chesapeake based DataLoaders.
 
         Args:
-            naip_root: directory containing NAIP data
-            chesapeake_root: directory containing Chesapeake data
             batch_size: The batch size to use in all created DataLoaders
             num_workers: The number of workers to use in all created DataLoaders
             patch_size: size of patches to sample
             **kwargs: Additional keyword arguments passed to
-                :class:`~torchgeo.datasets.NAIP` and
-                :class:`~torchgeo.datasets.Chesapeake13`
+                :class:`~torchgeo.datasets.NAIP` (prefix keys with naip_) and
+                :class:`~torchgeo.datasets.Chesapeake13` (prefix keys with chesapeake_)
```

Comment:
When I first added this I was lazy and didn't provide a way to separate these. This seems like a good solution to me.